index.html

```diff
@@ -139,13 +139,13 @@ <h2 class="card__content--heading">Tonic</h2>
           height="448">
         </div>
         <div class="card__content">
-          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <h1 class="card__content--heading">Multi-Post Stories</h1>
           <ul class="card__content--item1">
             <li class="card__item--heading">Canopy</li>
             <li> <img src="Images/Counter.png" alt="Counter"> Back End Dev</li>
             <li> <img src="Images/Counter.png" alt="Counter"> 2015</li>
           </ul>
-          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <h2 class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
```

Comment:
- [ ] Instead of using h2 here blush Please use `p` because this is the paragraph of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) +1,
- [ ] Kindly make sure you apply this to every heading with the class `card__content--desc` to meet the requirement :+1: